SCIBarcodeScanner/View/SCIBarcodeScannerView.swift

```diff
@@ -8,13 +8,40 @@ public protocol SCIBarcodeScannerViewDelegate {
     func sciBarcodeScannerPermissionMissing()
 }
 
+enum AlertStrings: String {
+    case title = "camera.authorization.title"
```

Comment:
Should we really use these MIOesque keys?

Reply:
I changed them to be general